drivers/clock_control/clock_stm32_ll_h5.c

```diff
@@ -367,11 +367,11 @@ static uint32_t get_vco_output_range(uint32_t vco_input_range)
 
 static void set_regu_voltage(uint32_t hclk_freq)
 {
-	if (hclk_freq < MHZ(80)) {
+	if (hclk_freq < MHZ(100)) {
```

Comment:
maybe `<=` will be better here, because for 100 MHz VOS3 can be used

Reply:
and VOS2 at 150MHz
and VOS1 at 200MHz